### ISSUE TITLE
[Bug]: Version on macOS 13shows "ERROR: Unable to retrieve store!"

### ISSUE THREAD

**Issue by ethan0418** [PlayCover/PlayCover#440]
### Describe the bug

/Library/Developer/CommandLineTools

Installing PlayTools
Copying PlayTools to Frameworks

ERROR: Unable to retrieve store!
Unable to retrieve store!
ERROR: Unable to retrieve store!
Unable to retrieve store!
ERROR: Unable to retrieve store!
Unable to retrieve store!
![image](https://user-images.githubusercontent.com/116659999/197810006-a8a0ca62-f5c7-4c16-8c07-cf14dc4b2c5b.png)

### Steps to reproduce

This accident occurred when I updated to macOS 13Reinstall but cannot resolve

### Expected behaviour

How can I solve

### Crash log

```shell
Version 13.0 (version 22A380)
/Library/Developer/CommandLineTools

Installing PlayTools
Copying PlayTools to Frameworks

ERROR: Unable to retrieve store!
Unable to retrieve store!
```

### What version of PlayCover are you using?

1.1.1

### What version of macOS are you using?

Ventura (macOS 13)

### Issue Language

- [X] Yes my issue is written in English

**Comment by ohaiibuzzle** [comment]
TLDR this is not a bug. We shut down the endpoints related to the old IPA download links and notices on the PlayCover website since we are not going to make use of it anymore, so clients that expects it to exist will throw an error. Please update to the 2.0 beta, or just spam click on the OK button

**Comment by ethan0418** [comment]
> 

thank you! I'll try

**Comment by tzm41** [comment]
How do I update to the 2.0 beta? The [Pre-release 2.0.0-alpha.1](https://github.com/PlayCover/PlayCover/releases/tag/2.0.0-alpha.1) only contains source code zip file.

**Comment by tzm41** [comment]
NVM found the nightly build.
